modules/ROOT/examples/configuration/sync-gateway-config.json

```diff
@@ -110,3 +110,371 @@
 }
 //
 // end::sync-function-config[]
+
+
+// tag::icr-simple-conflict-resolver[]
```

Comment:
It looks like these examples need to be updated to use the function(conflict) signature.